src/app.component.tsx

```diff
@@ -13,11 +11,21 @@ import classes from './app.module.scss'
 export function AppComponent() {
   const { t } = useTranslation()
   useDocumentTitle(t('label.title'))
-  const [isOpenRouteSelect, setIsOpenRouteSelect] = useState(false)
+  const [isOpenRouteSelect, setIsOpenRouteSelect] = useState<boolean>(false)
   // Maybe keep it in local storage?
   const [showUserLocation, setShowUserLocation] = useState<number | undefined>(undefined)
   const [selectedRoutes, setSelectedRoutes] = useSelectedRoutes()
-  const [openNote, setOpenNote] = useState(true)
+  const [firstVisit, setFirstVisit] = useState<boolean>(() => !areStoredRoutesEmpty())
```

Comment:
Thought about smth like this, but this way `Tooltip` switches between uncontrolled & controlled modes
```
<Tooltip
  title={t('label.pickRoute')}
  placement="left"
  {...firstVisit ? { open: true } : {}}
>...
```

Reply:
Let's keep it how is now.

Reply:
Also in most cases, there is no need for a gif preview :) Netlify takes care of deploying a frontend instance, so everyone can test the feature.

Reply:
Just in case ;)